.prettierignore

```diff
@@ -1,4 +1,8 @@
-app/assets/stylesheets/rails_admin/aristo
-app/assets/stylesheets/rails_admin/themes/cerulean
+coverage
+lib/generators/rails_admin/templates
```

Comment:
There's reason for it...

Some of files contained here (specifically speaking, they're `environment.js` and `webpack.config.js`) are need to be merged with existing ones which are created by Webpacker or Rails. And if a user already has some configurations, they need to be merged manually with the changes RailsAdmin need.

If we apply prettier to our templates and run RailsAdmin installer, what users get is like this:

```diff
% rails g rails_admin:install
...
    conflict  webpack.config.js
Overwrite /path/to/webpack.config.js? (enter "h" for help) [Ynaqdhm] d
--- /path/to/webpack.config.js 2021-12-04 13:53:59.000000000 +0900
+++ /path/to/webpack.config.js20211204-51180-i5ghjr    2021-12-04 13:54:22.000000000 +0900
@@ -1,18 +1,32 @@
-const path    = require("path")
-const webpack = require('webpack')
+const path = require("path");
+const webpack = require("webpack");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
...
```

 those diffs are unnecessary if we don't apply prettier. Then it looks like:

```diff
    conflict  webpack.config.js
Overwrite /path/to/webpack.config.js? (enter "h" for help) [Ynaqdhm] d
--- /path/to/webpack.config.js 2021-12-04 13:53:59.000000000 +0900
+++ /path/to/webpack.config.js20211204-51508-jf8k2a    2021-12-04 13:58:12.000000000 +0900
@@ -1,18 +1,32 @@
 const path    = require("path")
 const webpack = require('webpack')
+const MiniCssExtractPlugin = require("mini-css-extract-plugin")
...
```

much cleaner.
My intention is enabling users to install RailsAdmin with less work, instead of applying prettier to templates to get feeling of tidiness. Users can choose to apply prettier to their source code anyway, without hustle.